examples/workflow-count-resourcequota.yaml

```
@@ -17,7 +17,7 @@ metadata:
       <= v3.1 you can configure a global limit to the total number of running workflows in the cluster. This is suitable
       for single-tenancy set-up.
 
-      For multi-tenancy set-up, where each tenant own a single namespace, then in >= v3.1 you can globaly configure a
+      For multi-tenancy set-up, where each tenant own a single namespace, then in >= v3.1 you can globally configure a
```

```suggestion
      For multi-tenancy set-up, where each tenant owns a single namespace, then in >= v3.1 you can globally configure a
```

There's more typos on this line and in this comment too -- line 14 says "Workflowws".

although this entire comment could use a copy-edit to be simpler and more direct (per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language))